templates/exporter/licenses/nginx-prometheus-exporter/LICENSE

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
Apache License is not enough, we must to includes licensees of all dependencies that go binary is used.

Please simple copy licenses directory from NGINX.

Reply:
Done.